packages/react/src/components/carbon-components-react/UIShell/SideNavMenuSection.js

```diff
@@ -41,9 +41,10 @@ const SideNavMenuSection = ({
           'transitionend',
           function focus(event) {
             if (
-              event.propertyName === 'left' ||
-              event.propertyName === 'background-color' ||
-              event.propertyName === 'color'
+              focusElement &&
+              (event.propertyName === 'left' ||
+                event.propertyName === 'background-color' ||
+                event.propertyName === 'color')
             ) {
               focusElement.focus();
```

Comment:
```suggestion
              focusElement?.focus();
```

would this work as well?

Reply:
ah yes! 🙏 Thank you @emyarod, I forget about the optional chaining sometimes - looks so much cleaner!